lib/CodeGen/SelectionDAG/LegalizeDAG.cpp

```diff
@@ -3543,6 +3543,7 @@ void SelectionDAGLegalize::ExpandNode(SDNode *Node) {
     Tmp2 = Node->getOperand(0);
     Tmp3 = Node->getOperand(1);
     if (TLI.isOperationLegalOrCustom(DivRemOpc, VT) ||
+        TLI.preferIntDivRem() ||
```

Comment:
Do we really need a new virtual function for this? Isn't it enough to set `*DIV` and `*REM` to `Expand` and `*DIVREM` to `Custom`?